internal/store/lease.go

```diff
@@ -40,15 +40,15 @@ var (
 			metric.Gauge,
 			"",
 			wrapLeaseFunc(func(l *coordinationv1.Lease) *metric.Family {
-				labelKeys := []string{"owner_kind", "owner_name"}
+				labelKeys := []string{"owner_kind", "owner_name", "namespace", "lease_holder"}
 
 				owners := l.GetOwnerReferences()
 				if len(owners) == 0 {
 					return &metric.Family{
 						Metrics: []*metric.Metric{
 							{
 								LabelKeys:   labelKeys,
-								LabelValues: []string{"<none>", "<none>"},
+								LabelValues: []string{"<none>", "<none>", l.Namespace, *l.Spec.HolderIdentity},
```

Comment:
Can `*l.Spec.HolderIdentity` be nil in some cases?

Reply:
Yes HolderIdentity is optional. Thanks for catching this!